### ISSUE TITLE
Hide Redundancy

### ISSUE THREAD

**Issue by Erik262** [AdguardTeam/AdGuardHome#946]
### Prerequisites

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### Problem Description

I have a cam, which is sending every second, data to an website in my blocking list. This cause a huge redundancy statistics on the dashboard.

### Proposed Solution

The possibility to hide some client or ignore counting them

**Comment by ameshkov** [comment]
So the cam's DNS client ignores the blocked response's TTL?

Which rule blocks it? Could you please try blocking this domain with unspecified IP instead of NXDOMAIN? You can do it if you add smth like this to the custom rules:
`0.0.0.0 example.org`

**Comment by Erik262** [comment]
No, I put this rule self in my block list and everything works. Maybe this pic can help to understand what I mean.

![pic](https://user-images.githubusercontent.com/11977647/63493307-a0f74980-c4bb-11e9-8dbd-0f98b8d3ec0a.png)

While my IP Cam is sending data to a Chinese website (which I putted on my block list) the request count get higher and higher. So my statistics are not really usable. Thats why I'm asking for the feature to hide client or to ignore them counting in the statistics.

**Comment by ArchiveBase** [comment]
No ignore or excludes then here go https://github.com/AdguardTeam/AdGuardHome/issues/481#issuecomment-515673586 : Sort under **client range** so **the result** also applies to dashabord components means solve @Erik262 concern

**Comment by Ultrabenosaurus** [comment]
Being able to specify domains that are hidden from the Dashboard stats and Query Log, with a toggle switch in the top-right of both pages to include / exclude those domains at any time, would be amazing. 

Avast's anti-virus / internet security software has a few domains they ping regularly for "connectivity checks" and for the email anti-spam function. They ping the email one every 5 seconds as long as your device is on regardless of whether you're doing anything email related or not. Luckily the email one can be added to the `hosts` file without issue, so it doesn't impact my AGH stats yet, but Avast has an anti-tamper thing built-in to ensure other software cannot bypass, alter, or disable Avast. This includes removing most Avast-related domains from the `hosts` file, so my only choice is to add them as custom filters to AGH. 

Even without the email domain reaching AGH, just `analytics.ff.avast.com` alone accounts for a third to a half of all blocked requests in any given 24 hour period, significantly skewing my statistics and adding a lot of pointless entries to my query log and is constantly the number 1 for both the queried domains and blocked domains on my AGH Dashboard! 

I've also just discovered `ncc.avast.com` and added it to AGH, so I'm expecting my stats to get even less meaningful this time tomorrow. If it's pinged half as often as `analytics.ff.avast.com` then between them they'll account for over half of all blocked domains every day.

**Comment by ameshkov** [comment]
Marked as a feature request. However, I have no clear vision on how to implement this.

**Comment by ArchiveBase** [comment]
@ameshkov > However, I have no clear vision on how to implement this.
   
> in any given 24 hour period, significantly skewing my statistics and adding a lot of pointless entries to my query log and is constantly the number 1 for both the queried domains and blocked domains on my AGH Dashboard!

Add various sort methods. it is the best and easy solution for these type of feature requests without weaken AGHome. just simple data management ;) so users able to input certain domains to exclude and generate rest of statistics as desired (result apply to query log and dashboard)

**Comment by ameshkov** [comment]
I mean if this is just about query log, this is one thing. But if this is about filtering on a different level, and getting rid of it in the dashboard, it is a whole different story.

**Comment by ArchiveBase** [comment]
Yes but i see continuously receive similar feature requests time to time then i preferred to suggest most suitable solution to avoid break other scenarios.

**Comment by szolin** [comment]
I think we can add a new per-client configuration setting.  If it's on, we just don't include this client's IP addresses in statistics data that AGH web server sends to UI.  However, we should also provide a "red mark" somewhere in the clients table that would help admin not to forget about it.

**Comment by ArchiveBase** [comment]
https://github.com/AdguardTeam/AdGuardHome/issues/594#issuecomment-580248319

**Comment by WildByDesign** [comment]
I would also enjoy having the ability to exclude specific domains from the Statistics in the Dashboard.

For example, I have one domain name originating from my IPTV box that is accounting for something like 18% of overall DNS lookups on my entire network. So this naturally is skewing my Statistics and making them more difficult to understand.

**Comment by bruvv** [comment]
This is how pihole did it: https://github.com/pi-hole/FTL/pull/254/files and I also want to have this feature 👍 
( https://github.com/AdguardTeam/AdGuardHome/issues/1894 )

**Comment by rfgamaral** [comment]
I have the same problem and it«s even worse in my case since a specific internal domain accounts for more than 70% of all my DNS queries. It completely skews statistics and makes them hard to read.

For me, a simple "ignore this domain from all statistics" option would be more than enough :)
